DomainContent/Rust/DJ-Tools/DJ-Tools_Spawner.js

```diff
@@ -0,0 +1,560 @@
+        color = color || col(1, 1, 1);
+        userData = userData || {};
+        var props = {
+            name: name,
+            type: "Box",
+            position: pos,
+            locked: false,
+            dimensions: dim,
+            color: color,
+            visible: false,
+            collisionless: true,
+            userData: userData
+        };
+        var id = Entities.addEntity(props);
+        return id;
+    }
+
+    function makeZone(name, pos, dim, userData) {
+        name = name || 1;
+        dim = dim || vec(1, 1, 1);
+        userData = userData || {};
+        var props = {
+            name: name,
+            type: "Zone",
+            position: pos,
+            locked: false,
+            script: zoneScript + "?v=" + Date.now(),
+            dimensions: dim,
+            collisionless: true,
+            userData: userData
+        };
+        var id = Entities.addEntity(props);
+        return id;
+    }
+
+    function makeModel(name, pos, dim, rot, url, userData) {
+        name = name || "";
+        dim = dim || vec(1, 1, 1);
+        userData = userData || {};
+        var props = {
+            name: name,
+            type: "Model",
+            modelURL: url,
+            position: pos,
+            rotation: rot,
+            locked: false,
+            dimensions: dim,
+            collisionless: true,
+            userData: userData
+        };
+        var id = Entities.addEntity(props);
+        return id;
+    }
+
+    function makeParticle(name, pos, userData) {
+        name = name || "";
+        userData = userData || {};
+        var props = {
+            name: name,
+            locked: false,
+            position: pos,
+            serverScripts: particleScriptServer + "?v=" + Date.now(),
+            userData: userData
+        };
+        var finalParticleProps = Object.assign({}, particleBaseProps, props);
+        var id = Entities.addEntity(finalParticleProps);
+        return id;
+    } 
+
+    function makeLight(name, pos, dim, rot, color, isSpot, userData) {
+        name = name || "";
+        userData = userData || {};
+        var props = {
+            name: name,
+            position: pos,
+            dimensions: dim,
+            rotation: rot,
+            color: color,     
+            locked: false,
+            isSpotlight: isSpot,
+            serverScripts: lightScriptServer + "?v=" + Date.now(),
+            userData: userData
+        };
+        var finalLightProps = Object.assign({}, lightBaseProps, props);
+        var id = Entities.addEntity(finalLightProps);
+        return id;
+    }
+
+    // Procedural Functions
+    function deleteIfExists() {
+        entityNames.forEach(function (names) {
+            var found = Entities.findEntitiesByName(names, phlashProps[1].position, 20);
+            if (found.length === 1) {
+                Entities.deleteEntity(found[0]);
+            }
+        });
+    }
+
+    function createDebugCube() {
```

Comment:
There is a 'createDebugCube' and 'makeDebugCube', could they be consolidated or renamed to be more apparent as to the differences?

Reply:
Done.